src/WebListener.js

```diff
@@ -134,20 +141,23 @@ export default class WebListener {
 
   handleError = () => this.emitter.emit('error', new Error('WebSocket error'));
 
-  handleClose = () => this.emitter.emit('close');
+  handleClose = () => {
+    this.emitter.emit('close');
+    this.reconnect();
```

Comment:
So it'll emit close before reconnecting?

Reply:
Yes, we still will technically receive a close message prior to reconnecting.